bundles/org.openhab.binding.salus/src/main/resources/OH-INF/i18n/salus.properties

```diff
@@ -22,6 +22,7 @@ thing-type.config.salus.salus-aws-bridge.clientId.description = The app client I
 thing-type.config.salus.salus-aws-bridge.companyCode.label = Company Code
 thing-type.config.salus.salus-aws-bridge.group.aws.label = AWS
 thing-type.config.salus.salus-aws-bridge.group.aws.description = AWS Properties
+thing-type.config.salus.salus-aws-bridge.identityPoolId.label = Identity Pool ID
```

Comment:
This label was missing before? Does not seem to be related to this PR.

Reply:
Ive just run i8n for the plugin and this poped up